examples/code/ffi/posix.topscript

```diff
@@ -82,12 +83,6 @@ let gettimeofday' () =
   let secs = Signed.Long.(to_int (getf tv tv_sec)) in
   let usecs = Signed.Long.(to_int (getf tv tv_usec)) in
   Pervasives.(float secs +. float usecs /. 1000000.0) ;;
-[%%expect{|
-Characters 59-71:
-Error: Unbound value gettimeofday
-|}];;
+[%%expect ocaml {|val gettimeofday' : unit -> float = <fun>|}];;
 gettimeofday' () ;;
-[%%expect{|
-Characters 0-13:
-Error: Unbound value gettimeofday'
-|}];;
+[%%expect ocaml {|- : float = 1516746708.884176|}];;
```

Comment:
Note that this is non-deterministic. Maybe later it would make sense to wrap a different system call whose output we can arrange to be deterministic?

Reply:
Yeah, I've made a note of this when doing my Ctypes refresh